docs/host/host.md

```diff
@@ -33,30 +33,122 @@ Cordoning a node marks it as unschedulable. This feature is useful for performin
 
 ## Deleting a Node
 
-Deleting a node is done in two phases:
+:::caution
 
-1. Delete the node from Harvester
-    - Go to the **Hosts** page
-    - On the node you want to modify, click **⋮ > Delete**
+Before removing a node from a Harvester cluster, determine if the remaining nodes have enough computing and storage resources to take on the workload from the node to be removed. Check the following:
 
-2. Uninstall RKE2 from the node
-    - Login to the node as root
-    - Run `rke2-uninstall.sh` to delete the whole RKE2 service.
+- Current resource usage in the cluster (on the **Hosts** screen of the Harvester UI)
+- Ability of the remaining nodes to maintain enough replicas for all volumes
 
-:::caution
+If the remaining nodes do not have enough resources, VMs might fail to start and volumes might degrade when you remove a node.
```

Comment:
Did it mean the VM failed to migrate?
The running VM did not need to start again.

Reply:
I think it's **VMs may fail to migrate due to a shortage of storage**?

Reply:
Thanks, add in https://github.com/harvester/docs/pull/507/commits/889a4a192ba3ecbb3c71addc737f8d512ed8080e

Reply:
@WebberHuang1118 it's not entirely just storage, could be cpu/memory. Thus I use the general term resource here.